core/domain/event_services_test.py

```diff
@@ -237,6 +237,8 @@ def _get_all_python_files(self):
             if not (directory.startswith('./core/') or
                     directory.startswith('./extensions/')):
                 continue
+            if directory.startswith('./core/tests/linter_tests'):
+                continue
```

Comment:
This (as well as the other two files) search for all the python files in core/ or extensions/, so we have to ignore linter_tests since it contains "mock" python files used for testing.